### DISCUSSION TITLE
RiMusic app design concept

### DISCUSSION THREAD

**Discussion by fergassss1** [fast4x/RiMusic#3928] [Show and tell]
Made in canva, and its my first time trying this!
Maybe something is just not too good but ill learn over time, and yeah i used memes for the placeholder images i didnt have better imgs...
![Song name_20241011_211140_0000](https://github.com/user-attachments/assets/532ef272-1ec7-4b1f-90d3-510e1b1d772c)

**Comment by fergassss1** [comment]
making another one by start on Figma, because its gonna be better
![Frame 3 (1)](https://github.com/user-attachments/assets/b65df3aa-89dd-4d0c-9cce-d077ce5428a2)

**Comment by fast4x** [comment]
Thanks, but you think that new is better then  actual?

**Reply by fergassss1** [reply]
> Thanks, but you think that new is better then actual?

i like both equally im just trying out app designing and recreating a modern look
when u play around with settings id say u get the best looks

**Comment by fergassss1** [comment]
![190 sin título_20241124155333](https://github.com/user-attachments/assets/8769e1a9-a6bf-4c6e-ad17-51ce3c12e21a)
![189 sin título_20241124134007](https://github.com/user-attachments/assets/226860ae-4e09-4b36-8caf-1788bde58abe)
![188 sin título_20241123204703](https://github.com/user-attachments/assets/bc239427-8691-4e1e-9bb3-4d489f0f5433)
